build.ps1

```diff
@@ -673,61 +748,38 @@ function Build-SourceKitLspX64
 
 #-------------------------------------------------------------------
 
-# preflight
+# Compilers
 Invoke-VsDevShell $ArchX64
-Build-ToolchainX64
-Build-LlvmX64
+Build-Toolchain $ArchX64
+Build-LLVM $ArchX64
 
-# Windows x64 Build
-Invoke-VsDevShell $ArchX64
-Build-ZLib $ArchX64
-Build-LibXml2 $ArchX64
-Build-Curl $ArchX64
-Build-Icu $ArchX64
-Build-SwiftRuntime $ArchX64
-Build-SwiftCorelibsLibdispatch $ArchX64
-Build-SwiftCorelibsFoundation $ArchX64
-Build-SwiftCorelibsXCTest $ArchX64
-
-if (-not $OnlyX64)
+foreach ($Arch in $ArchX64,$ArchX86,$ArchARM64)
```

Comment:
It would be nice to keep supporting `-OnlyX64` or an equivalent for local builds. In fact I'd vote for requiring `-X86 -X64 -Arm64` for building all platforms and putting the burden of passing the args on the CI automation.

Reply:
`-OnlyX64` is too ambiguous as are the switches you suggest.  The default should be to assume that you will build all the architectures, because there are differences in between them, and that is a requirement for development.  That is just the cost of working on the toolchain.

Eventually, I would be okay with `-hosts X64 -sdks X64` as an option, with the default being `-hosts X64;ARM64 -sdks X64;X86;ARM64` and possibly restoration of the ARM SDK as well.